### ISSUE TITLE
Add alpine and RHEL as runtime targets for testing to ensure compatibility with common environments

### ISSUE THREAD

**Issue by iokiwi** [tgenv/tgenv#14]

**Comment by weyderfs** [comment]
@iokiwi Is there a way to test locally? My worry it's about broken the `main` tests pipeline.

**Comment by iokiwi** [comment]
Yes and we can also reconfigure the github actions to run the same tests on branches/pull requests before merging to main. I didn't notice that the current gitlab action workflow copied from tgenv ran different tests on main vs on a branch but will account for that going forward.

**Comment by weyderfs** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Testes TGenv RHEL + Alpine](https://trello.com/c/mLjHAPOU/30-testes-tgenv-rhel-alpine)
